js/decklist/main.js

```diff
@@ -45,6 +45,7 @@ function pdfChangeWait() {
   decklistChangeTimer = setTimeout(function() {
     const parsedInput = Decklist.parse();
     validateInput(parsedInput);
+    $('#permalink').attr("href", generatePermalink());
```

Comment:
Rather than setting the link's (button's) `href` attribute, [why not push the URL as a new history state](https://developer.mozilla.org/en-US/docs/Web/API/History_API#Example_of_pushState()_method)?

Reply:
Oh, thanks! I haven't done much JS/web stuff at all; had a vague idea this was possible but haven't actually done it before.  Will give it a shot.

Reply:
Alternatively (or in addition!) you could try copying the URL to the clipboard.